druid-shell/src/platform/x11/window.rs

```diff
@@ -224,6 +224,14 @@ impl WindowBuilder {
                 | EventMask::PointerMotion,
         );
 
+        // Conversion factor for scale (units = mm/pixels)
+        const FACTOR: f64 = 25.4 / 96.0;
+        let scale = Scale::new(
+            FACTOR * screen.width_in_pixels as f64 / screen.width_in_millimeters as f64,
+            FACTOR * screen.height_in_pixels as f64 / screen.height_in_millimeters as f64,
+        );
+        dbg!(scale);
```

Comment:
```suggestion
```

Reply:
Not so simple... the scale value is wrong.